youtube_dl/extractor/radiocloud.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import re
+
+from .common import InfoExtractor
+
+from ..utils import (
+    ExtractorError,
+    extract_attributes,
+    get_elements_by_class,
+)
+
+
+class RadioCloudIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radiocloud\.jp/archive/.*?content_id=(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://radiocloud.jp/archive/dc?content_id=15326',
+        'md5': 'd7bcf8b2877e694543a1f462b458a665',
+        'info_dict': {
+            'id': '15326',
+            'ext': 'm4a',
+            'title': '「オープニング」 ',
+            'description': '「“フィギュアスケートは人生”引退会見で浅田真央さんは何を語ったのか？」スポーツライター・青嶋ひろのさんが解説！',
+        }
```

Comment:
Interesting, it seems like for people without an account the page loads content a week or two into the past. For people who are logged in with an account the page loads content a month into the past.

I'll try to work out if I can find a way to load user restricted content without authentication. If that fails, I'll implement authentication (but I'll make it into its own commit instead of amending the current  commit like for the other fixes).

I'm not sure if there is any access to content older than a month. I'll fetch the extracted URL for a recording about to expire and see if I can still access it after it expires.

If I can't find a way to download content which the website does not give direct access to even for logged in users, what should I do for tests which have an "expiration date" so to speak?